templates/lib/systemd/system/atom-worker.service

```diff
@@ -24,7 +26,8 @@ ExecStart=/usr/bin/php{{ php_version }} -d memory_limit=-1 -d error_reporting="E
 {% endif %}
 {% endif %}
 KillSignal=SIGTERM
-Restart=no
+Restart=on-failure
+RestartSec=5
```

Comment:
I like @jraddaoui suggestion to bump this up to `RestartSec=10` to allow a bit more time between restarts and allow more time for a transitory failure cause to expire.